tests/models/test_momentum_balance.py

```diff
@@ -123,22 +123,22 @@ def test_unit_conversion(units, uy_north):
     """
     solid = pp.SolidConstants(pp.solid_values.extended_granite_values_for_testing)
 
-    params = {
+    model_params = {
         "times_to_export": [],  # Suppress output for tests
         "fracture_indices": [0, 1],
         "cartesian": True,
         "uy_north": uy_north,
         "material_constants": {"solid": solid},
     }
-    reference_params = copy.deepcopy(params)
+    model_reference_params = copy.deepcopy(model_params)
```

Comment:
Why not `reference_model_params`? Wouldn't this better follow the style of calling the `params`, `model_params`?

Reply:
I would suggest: model_params_ref